docs/basic-features/environment-variables.md

```diff
@@ -152,7 +152,7 @@ export default async () => {
 
 ## Environment Variable Load Order
 
-Depending on the environment (as set by `NODE_ENV`), Environment Variables are loaded from the following sources in top-to-bottom order. In all environments, the existing `env` is not overridden by following sources:
+Depending on the environment (as set by `NODE_ENV`), Environment Variables are defined from the following sources. The sources are ordered in descending priority, so environment variables from source (4) are loaded first, then 3 will override, then 2 will override, and finally 1 has the highest priority and will override all others. In all environments, the existing system `env` has the highest priority and is not overridden.
```

Comment:
I'm not sure this is more clear as we don't override already loaded env although this does seem tricky to word 😅 

The related order we load can be seen here https://github.com/vercel/next.js/blob/af86ca08e4363f2bbc10e4bd94a76b4f62b106c2/packages/next-env/index.ts#L83-L89 

We load each env file and as long as a key it defines isn't already set in process.env it will be set

Reply:
Yea, I was pondering this exact same case ^
Any suggestions?

Reply:
I think the changes were more clear in https://github.com/vercel/next.js/pull/36967/commits/2aea8c51a7ab7367d1baa654f5fc58e72c7e0b66